.golangci.yml

```diff
@@ -7,5 +7,17 @@ run:
 linters:
   enable:
     - gofumpt
+    - gosec
     - errorlint
     - unconvert
+
+linters-settings:
+  gosec:
+    confidence: medium
+    severity: medium
+    excludes:
+      - G301
```

Comment:
Bit on the fence if we should disable this one, or instead add `nolint` comments. Admitted, this check _does_ produce many "false positives", but I also know of some advisories that were related to incorrect permissions set (don't recall from the top of my head if those were in runc, but  at least in moby/docker and containerd).

Reply:
Okay, I'll add nosecs. But, be aware that I will not be able to evaluate if they are correct.